### ISSUE TITLE
Redis CLI requires `COMMAND` command on connect

### ISSUE THREAD

**Issue by blainsmith** [tidwall/redcon#32]
While this is not technically a limitation of this library, but as I was implementing a server with redcon I noticed that my local `redis-cli` client v5.0.7 didn't seem to work with the redcon-based server I made. After doing some digging I noticed that the `redis-cli` issues a [`COMMAND` command](https://redis.io/commands/command) after a connection is established and it expects a response to work. Once I responded to that with redcon the official `redis-cli` worked fine. Not sure if you want to add this as a not in the README if people attempt to use it and it doesn't work as expected right away.

**Comment by tidwall** [comment]
I'm not able to reproduce. I just built `5.0.7` directly from `github.com/antirez/redis` and ran the `redis-cli` against the `go example/clone.go` in the redcon repo and it worked with errors or warnings.

These were my steps.

**Terminal 1**

Redcon clone example

```
$ go get github.com/tidwall/redcon/example
$ example
2020/06/20 16:41:08 started server at :6380
```

**Terminal 2**

Build and run redis-cli 5.0.7

```
$ git clone https://github.com/antirez/redis
$ cd redis
$ git checkout 5.0.7
$ make
$ src/redis-cli -p 6380
127.0.0.1:6380>
```

This was from my Mac.

**Comment by tidwall** [comment]
I wonder if there's a build setting or a flag that adds that feature. 
Did you build your redis-cli or was it installed from a package manager?

**Comment by blainsmith** [comment]
Interesting I installed it via `apt` on Ubuntu. I will do some more digging based on what you found and see if I can come to a resolution. I had the same result you did with your clone example and my current `redis-cli`. Stay tuned!

**Comment by blainsmith** [comment]
Ah I found it. It wasn't the `COMMAND` command. If you remove the `default` case in the command switch `redis-cli` will not provide you with the prompt. I was able to reproduce this if you [remove these lines](https://github.com/tidwall/redcon/blob/master/example/clone.go#L20-L21) from the clone example.

Missing `default` case
```
> redis-cli -p 6380           

```

With `default` case
```
> redis-cli -p 6380                                 
127.0.0.1:6380>
```
